miniupnpd/configure

```diff
@@ -451,23 +524,38 @@ case $OS_NAME in
 			V6SOCKETS_ARE_V6ONLY=`$(find /sbin /bin /usr/sbin /usr/bin -name sysctl) -n net.ipv6.bindv6only`
 			fi
 		fi
+
+		if ldd --version | grep -q GLIBC; then
```

Comment:
hum that part wont work for cross builds :(

Reply:
Ideally it needs a compile test, but isn't it exactly the way it was before, just in the Makefile instead of the configure script?

Reply:
yes, your commit is OK, it was exactly the way it was before.

I'm looking to improve that :)

Reply:
I guess I should try something like `echo "#include <features.h>" | cc -dM -E -` 
and look for `__GLIBC__` / `__GLIBC_MINOR__`

Reply:
Test-compile something with a call to `clock_gettime()` and see if it links :-)

I'll see if I can stick that into configure.

Reply:
Or maybe `clock_gettime()` is included in every recent libc :wink: 

it looks like glibc 2.16 is from 2012...

Reply:
The question here is whether `clock_gettime()` is included in `-lc` or whether it needs a `-lrt`.

I can drop this altogether if compatibility with pre-2012 systems is not a concern.

Reply:
According to the [glibc 2.17 announcement](https://sourceware.org/legacy-ml/libc-announce/2012/msg00001.html) I feel that using `clock_gettime()` on glibc 2.16 and before is not worth it : 
> * The `clock_*` suite of functions (declared in <time.h>) is now available
  directly in the main C library.  Previously it was necessary to link with
  -lrt to use these functions.  This change has the effect that a
  single-threaded program that uses a function such as `clock_gettime` (and 
  is not linked with -lrt) will no longer implicitly load the pthreads
  library at runtime and so will not suffer the overheads associated with
  multi-thread support in other code such as the C++ runtime library.

Reply:
cleaned there : 8e5e40a8ce9f38400f9779ab74cca7ef7d95ab2a